app/vmalert/main.go

```diff
@@ -16,22 +22,35 @@ import (
 var (
 	configPath     = flag.String("config", "config.yaml", "Path to alert configuration file")
 	httpListenAddr = flag.String("httpListenAddr", ":8880", "Address to listen for http connections")
+
+	datasourceURL      = flag.String("datasource.url", "", "Victoria Metrics or VMSelect url. Required parameter. e.g. http://127.0.0.1:8428")
```

Comment:
flag names inconsistency - dots with camel case.

Reply:
naming schema has been taken from `vmalert`

```
go run app/vmalert/main.go --help

  -config string
    	Path to alert configuration file (default "config.yaml")
  -datasource.basicAuth.password string
    	Optional basic auth password to use for -datasource.url
  -datasource.basicAuth.username string
    	Optional basic auth username to use for -datasource.url
  -datasource.url string
    	Victoria Metrics or VMSelect url. Required parameter. e.g. http://127.0.0.1:8428
  -enableTCP6
    	Whether to enable IPv6 for listening and dialing. By default only IPv4 TCP is used
  -envflag.enable
    	Whether to enable reading flags from environment variables additionally to command line. Command line flag values have priority over values from environment vars. Flags are read only from command line if this flag isn't set
  -evaluationInterval duration
    	How often to evaluate the rules. Default 1m (default 1m0s)
  -http.disableResponseCompression
    	Disable compression of HTTP responses for saving CPU resources. By default compression is enabled to save network bandwidth
  -httpAuth.password string
    	Password for HTTP Basic Auth. The authentication is disabled if -httpAuth.username is empty
  -httpAuth.username string
    	Username for HTTP Basic Auth. The authentication is disabled if empty. See also -httpAuth.password
  -httpListenAddr string
    	Address to listen for http connections (default ":8880")
  -loggerFormat string
    	Format for logs. Possible values: default, json (default "default")
  -loggerLevel string
    	Minimum level of errors to log. Possible values: INFO, ERROR, FATAL, PANIC (default "INFO")
  -loggerOutput string
    	Output for the logs. Supported values: stderr, stdout (default "stderr")
  -memory.allowedPercent float
    	Allowed percent of system memory VictoriaMetrics caches may occupy (default 60)
  -metricsAuthKey string
    	Auth key for /metrics. It overrides httpAuth settings
  -pprofAuthKey string
    	Auth key for /debug/pprof. It overrides httpAuth settings
  -provider.url string
    	Prometheus alertmanager url. Required parameter. e.g. http://127.0.0.1:9093
  -tls
    	Whether to enable TLS (aka HTTPS) for incoming requests. -tlsCertFile and -tlsKeyFile must be set if -tls is set
  -tlsCertFile string
    	Path to file with TLS certificate. Used only if -tls is set. Prefer ECDSA certs instead of RSA certs, since RSA certs are slow
  -tlsKeyFile string
    	Path to file with TLS key. Used only if -tls is set
  -version
    	Show VictoriaMetrics version
```